# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# Andrew Martynov <andrewm@inventa.ru>, 2006
# triplepointfive <iya777@yandex.ru>, 2012
# Yulia <ypoyarko@redhat.com>, 2006, 2008, 2009, 2010, 2012
# Yulia <yulia.poyarkova@redhat.com>, 2013
# Игорь Горбунов <igor.gorbounov@gmail.com>, 2013
# yuliya <ypoyarko@redhat.com>, 2016. #zanata
# Vit Mojzis <vmojzis@redhat.com>, 2017. #zanata
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2017-11-20 17:16+0100\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"PO-Revision-Date: 2017-11-20 11:36-0500\n"
"Last-Translator: Copied by Zanata <copied-by-zanata@zanata.org>\n"
"Language-Team: Russian (http://www.transifex.com/projects/p/fedora/language/"
"ru/)\n"
"Language: ru\n"
"Plural-Forms: nplurals=3; plural=(n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
"n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2);\n"
"X-Generator: Zanata 3.9.6\n"

#: ../src/allow_anon_write.py:27
msgid ""
"\n"
"    SELinux policy is preventing an httpd script from writing to a public\n"
"    directory.\n"
"    "
msgstr ""
"\n"
"    Политика SELinux предотвращает выполнение записи в общий каталог запись "
"сценарием httpd.\n"
"    "

#: ../src/allow_anon_write.py:32
msgid ""
"\n"
"    SELinux policy is preventing an httpd script from writing to a public\n"
"    directory.  If httpd is not setup to write to public directories, this\n"
"    could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    Политика SELinux предотвращает выполнение записи в общий каталог\n"
"    сценарием httpd. Если httpd не настроен на запись в общие каталоги, \n"
"    это может служить индикатором попытки вторжения.\n"
"    "

#: ../src/allow_anon_write.py:38
msgid ""
"\n"
"    If httpd scripts should be allowed to write to public directories you "
"need to turn on the $BOOLEAN boolean and change the file context of the "
"public directory to public_content_rw_t.  Read the httpd_selinux\n"
"    man page for further information:\n"
"    \"setsebool -P $BOOLEAN=1; chcon -t public_content_rw_t <path>\"\n"
"    You must also change the default file context labeling files on the "
"system in order to preserve public directory labeling even on a full relabel."
"  \"semanage fcontext -a -t public_content_rw_t <path>\"\n"
"    "
msgstr ""
"\n"
"    Для разрешения сценариям httpd выполнения записи в общие каталоги "
"необходимо установить значение $BOOLEAN и изменить контекст общего\n"
"    каталога на public_content_rw_t. За подробной информацией обратитесь к "
"странице помощи httpd_selinux.\n"
"    «setsebool -P $BOOLEAN=1; chcon -t public_content_rw_t <путь>»\n"
"    Также необходимо изменить исходные контексты файлов в системе так, чтобы "
"они не изменялись при повторной разметке. \n"
"    «semanage fcontext -a -t public_content_rw_t <путь>»\n"
"    "

#: ../src/allow_anon_write.py:44
msgid ""
"If you want to allow $SOURCE_PATH to be able to write to shared public "
"content"
msgstr ""

#: ../src/allow_anon_write.py:45
msgid ""
"you need to change the label on $TARGET_PATH to public_content_rw_t, and "
"potentially turn on the allow_httpd_sys_script_anon_write boolean."
msgstr ""
"необходимо изменить метку $TARGET_PATH на public_content_rw_t и, возможно, "
"установить переменную allow_httpd_sys_script_anon_write."

#: ../src/allow_execheap.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from changing the access\n"
"    protection of memory on the heap.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает $SOURCE_PATH изменять\n"
"    защиту доступа памяти кучи.\n"
"    "

#: ../src/allow_execheap.py:32
msgid ""
"\n"
"    The $SOURCE application attempted to change the access protection of "
"memory on\n"
"    the heap (e.g., allocated using malloc).  This is a potential security\n"
"    problem.  Applications should not be doing this. Applications are\n"
"    sometimes coded incorrectly and request this permission.  The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">SELinux "
"Memory Protection Tests</a>\n"
"    web page explains how to remove this requirement.  If $SOURCE does not "
"work and\n"
"    you need it to work, you can configure SELinux temporarily to allow\n"
"    this access until the application is fixed. Please file a bug\n"
"    report against this package.\n"
"    "
msgstr ""
"\n"
"    Приложение $SOURCE попыталось изменить защиту доступа памяти кучи\n"
"    (выделенной с помощью malloc), что представляет потенциальную проблему\n"
"    с точки зрения безопасности. Обычно приложения не должны пытаться это "
"сделать.\n"
"    Иногда вследствие ошибок кода приложения могут потребовать такой доступ.\n"
"    Документ\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">Тесты "
"защиты памяти SELinux</a>\n"
"    объясняет, как отказаться от использования такой возможности. Если "
"$SOURCE не работает, а\n"
"    вам срочно необходимо данное приложение, можно временно разрешить\n"
"    доступ пока приложение не будет исправлено. Пожалуйста, создайте\n"
"    запрос в Bugzilla для этого пакета.\n"
"    "

#: ../src/allow_execheap.py:44
msgid ""
"\n"
"    If you want $SOURCE to continue, you must turn on the\n"
"    $BOOLEAN boolean.  Note: This boolean will affect all applications\n"
"    on the system.\n"
"    "
msgstr ""
"\n"
"    Для продолжения $SOURCE необходимо установить\n"
"    логическую переменную $BOOLEAN. Замечание: Ее значение окажет\n"
"    влияние на все приложения системы.\n"
"    "

#: ../src/allow_execheap.py:50
msgid ""
"If you do not think $SOURCE_PATH should need to map heap memory that is both "
"writable and executable."
msgstr ""

#: ../src/allow_execheap.py:51
msgid "you need to report a bug. This is a potentially dangerous access."
msgstr ""
"следует создать отчет об ошибке. Это обращение представляет риск "
"безопасности."

#: ../src/allow_execheap.py:52 ../src/allow_execstack.py:91
#: ../src/kernel_modules.py:46 ../src/mmap_zero.py:48
#: ../src/selinuxpolicy.py:46 ../src/setenforce.py:29 ../src/setenforce.py:50
#: ../src/sys_module.py:30 ../src/sys_module.py:48
msgid "Contact your security administrator and report this issue."
msgstr "О проблеме сообщите администратору."

#: ../src/allow_execmod.py:29
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from loading $TARGET_PATH which "
"requires text relocation.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает $SOURCE_PATH загружать $TARGET_PATH.\n"
"    Загрузка $TARGET_PATH требует перемещения текста.\n"
"    "

#: ../src/allow_execmod.py:33
msgid ""
"\n"
"    The $SOURCE application attempted to load $TARGET_PATH which\n"
"    requires text relocation.  This is a potential security problem.\n"
"    Most libraries do not need this permission. Libraries are\n"
"    sometimes coded incorrectly and request this permission.  The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">SELinux "
"Memory Protection Tests</a>\n"
"    web page explains how to remove this requirement.  You can configure\n"
"    SELinux temporarily to allow $TARGET_PATH to use relocation as a\n"
"    workaround, until the library is fixed. Please file a\n"
"bug report.\n"
"    "
msgstr ""
"\n"
"    Приложение $SOURCE попыталось загрузить $TARGET_PATH, что\n"
"    требует перемещения текста. Это представляет потенциальную проблему с\n"
"    точки зрения безопасности. Иногда библиотеки\n"
"    написаны некорректно и требуют разрешения на перемещение. Документ\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">Тесты "
"защиты памяти SELinux</a>\n"
"    объясняет, как отказаться от использования такой возможности. Можно "
"настроить\n"
"    SELinux так, чтобы временно разрешить $TARGET_PATH использовать "
"перемещение,\n"
"    пока библиотека не будет исправлена. Пожалуйста, создайте \n"
"    запрос в Bugzilla для этого пакета.\n"
"    "

#: ../src/allow_execmod.py:45
msgid ""
"\n"
"    The $SOURCE application attempted to load $TARGET_PATH which\n"
"    requires text relocation.  This is a potential security problem.\n"
"    Most libraries should not need this permission.   The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">\n"
"    SELinux Memory Protection Tests</a>\n"
"    web page explains this check.  This tool examined the library and it "
"looks\n"
"    like it was built correctly. So setroubleshoot can not determine if this\n"
"    application is compromised or not.  This could be a serious issue. Your\n"
"    system may very well be compromised.\n"
"\n"
"    Contact your security administrator and report this issue.\n"
"\n"
"    "
msgstr ""

#: ../src/allow_execmod.py:62
msgid ""
"\n"
"    If you trust $TARGET_PATH to run correctly, you can change the\n"
"    file context to textrel_shlib_t. \"chcon -t textrel_shlib_t\n"
"    '$TARGET_PATH'\"\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"textrel_shlib_t '$FIX_TARGET_PATH'\"\n"
"\n"
"    "
msgstr ""
"\n"
"    Если вы доверяете $TARGET_PATH, можно изменить контекст\n"
"    файла на textrel_shlib_t. «chcon -t textrel_shlib_t \n"
"    $TARGET_PATH»\n"
"    Также необходимо изменить контексты файлов в системе, используемые по "
"умолчанию, чтобы они сохранились даже при повторной разметке. \n"
"    «semanage fcontext -a -t textrel_shlib_t '$FIX_TARGET_PATH'»\n"
"\n"
"    \n"
"    "

#: ../src/allow_execstack.py:60
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from making the program stack "
"executable.\n"
"    "
msgstr ""
"\n"
"   SELinux запрещает $SOURCE_PATH определять программный стек как "
"исполняемый.\n"
"    "

#: ../src/allow_execstack.py:64
msgid ""
"\n"
"    The $SOURCE application attempted to make its stack\n"
"    executable.  This is a potential security problem.  This should\n"
"    never ever be necessary. Stack memory is not executable on most\n"
"    OSes these days and this will not change. Executable stack memory\n"
"    is one of the biggest security problems. An execstack error might\n"
"    in fact be most likely raised by malicious code. Applications are\n"
"    sometimes coded incorrectly and request this permission.  The\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">SELinux "
"Memory Protection Tests</a>\n"
"    web page explains how to remove this requirement.  If $SOURCE does not\n"
"    work and you need it to work, you can configure SELinux\n"
"    temporarily to allow this access until the application is fixed. Please\n"
"file a bug report.\n"
"    "
msgstr ""
"\n"
"    Приложение $SOURCE попыталось сделать свой стек\n"
"    исполняемым, что представляет потенциальную угрозу безопасности.\n"
"    В этом не должно быть необходимости. \n"
"    Память стека не является исполняемой в большинстве ОС\n"
"    на сегодняшний день, и в будущем это не изменится.\n"
"    Исполняемый стек представляет собой исключительно серьезную проблему "
"безопасности.\n"
"    Ошибка execstack может быть вызвана вредоносными программами.\n"
"    Иногда приложения, написанные\n"
"    некорректно, могут запросить это разрешение. Документ\n"
"    <a href=\"http://people.redhat.com/drepper/selinux-mem.html\">Тесты "
"защиты памяти SELinux</a>\n"
"    объясняет, как исключить использование такой возможности. Если $SOURCE "
"не\n"
"    работает, когда это необходимо, можно настроить SELinux так, \n"
"    чтобы временно разрешить такую возможность, пока приложение не будет "
"исправлено. Пожалуйста, \n"
"    создайте запрос в Bugzilla для этого пакета.\n"
"    "

#: ../src/allow_execstack.py:79
msgid ""
"\n"
"    Sometimes a library is accidentally marked with the execstack flag,\n"
"    if you find a library with this flag you can clear it with the\n"
"    execstack -c LIBRARY_PATH.  Then retry your application.  If the\n"
"    app continues to not work, you can turn the flag back on with\n"
"    execstack -s LIBRARY_PATH.\n"
"    "
msgstr ""
"\n"
"Иногда для библиотеки может быть случайно выбран флаг execstack.\n"
"Его  можно снять, выполнив execstack -c LIBRARY_PATH.\n"
"После этого перезапустите приложение.\n"
"Если это не помогло, снова установите флаг: execstack -s LIBRARY_PATH. "

#: ../src/allow_execstack.py:89
msgid ""
"If you do not think $SOURCE_PATH should need to map stack memory that is "
"both writable and executable."
msgstr ""

#: ../src/allow_execstack.py:90
msgid "you need to report a bug. \n"
"This is a potentially dangerous access."
msgstr ""
"следует создать отчет об ошибке. \n"
"Это обращение представляет риск безопасности."

#: ../src/allow_execstack.py:99
#, python-format
msgid "If you believe that \n"
"%s\n"
"should not require execstack"
msgstr ""

#: ../src/allow_execstack.py:108
#, python-format
msgid ""
"you should clear the execstack flag and see if $SOURCE_PATH works correctly.\n"
"Report this as a bug on %s.\n"
"You can clear the exestack flag by executing:"
msgstr ""
"следует очистить флаг execstack и убедиться что $SOURCE_PATH функционирует "
"корректно.\n"
"Сообщите об этой ошибке в %s.\n"
"Чтобы очистить флаг exestack выполните:"

#: ../src/allow_execstack.py:118
#, python-format
msgid "execstack -c %s"
msgstr "execstack -c %s"

#: ../src/allow_ftpd_use_cifs.py:29
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a CIFS "
"filesystem.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил доступ $ACCESS демона ftp к файловой системе CIFS.\n"
"    "

#: ../src/allow_ftpd_use_cifs.py:33
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a CIFS "
"filesystem.\n"
"    CIFS (Comment Internet File System) is a network filesystem similar to\n"
"    SMB (<a href=\"http://www.microsoft.com/mind/1196/cifs.asp\">http://www."
"microsoft.com/mind/1196/cifs.asp</a>)\n"
"    The ftp daemon attempted to read one or more files or directories from\n"
"    a mounted filesystem of this type.  As CIFS filesystems do not support\n"
"    fine-grained SELinux labeling, all files and directories in the\n"
"    filesystem will have the same security context.\n"
"\n"
"    If you have not configured the ftp daemon to read files from a CIFS "
"filesystem\n"
"    this access attempt could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил демону ftp доступ $ACCESS к файлам в файловой системе "
"CIFS.\n"
"    CIFS (Comment Internet File System) — сетевая файловая система,\n"
"   аналогичная SMB (<a href=\"http://www.microsoft.com/mind/1196/cifs."
"asp\">http://www.microsoft.com/mind/1196/cifs.asp</a>).\n"
"    Демон ftp попытался прочесть данные из смонтированной системы этого типа."
"\n"
"    Так как CIFS не поддерживает низкоуровневое маркирование SELinux, \n"
"    все файлы и каталоги имеют один и тот же контекст безопасности.\n"
"     \n"
"    Если вы не настраивали демон ftp на чтение файлов из CIFS,\n"
"    попытка получения доступа может служить сигналом вторжения.\n"
"    "

#: ../src/allow_ftpd_use_cifs.py:46
msgid ""
"\n"
"    Changing the \"$BOOLEAN\" boolean to true will allow this access:\n"
"    \"setsebool -P $BOOLEAN=1.\"\n"
"    "
msgstr ""
"\n"
"    Установка логического значения «$BOOLEAN» в TRUE разрешит доступ:\n"
"    «setsebool -P $BOOLEAN=1»\n"
"    "

#: ../src/allow_ftpd_use_cifs.py:53
msgid ""
" Changing the \"$BOOLEAN\" and\n"
"    \"$WRITE_BOOLEAN\" booleans to true will allow this access:\n"
"    \"setsebool -P $BOOLEAN=1 $WRITE_BOOLEAN=1\".\n"
"    warning: setting the \"$WRITE_BOOLEAN\" boolean to true will\n"
"    allow the ftp daemon to write to all public content (files and\n"
"    directories with type public_content_t) in addition to writing to\n"
"    files and directories on CIFS filesystems.  "
msgstr ""
" Установка логических значений «$BOOLEAN» и\n"
"    «$WRITE_BOOLEAN» откроет доступ:\n"
"    «setsebool -P $BOOLEAN=1 $WRITE_BOOLEAN=1».\n"
"    Предупреждение: установка «$WRITE_BOOLEAN» в TRUE\n"
"    позволит ftp-демону осуществлять запись во все общие ресурсы\n"
"    (файлы и каталоги типа public_content_t) помимо записи в\n"
"    файлы и каталоги в файловых системах CIFS."

#: ../src/allow_ftpd_use_cifs.py:62
msgid "If you want to allow ftpd to write to cifs file systems"
msgstr ""

#: ../src/allow_ftpd_use_cifs.py:63 ../src/allow_ftpd_use_nfs.py:65
msgid "you must tell SELinux about this"
msgstr "необходимо сообщить об этом в SELinux"

#: ../src/allow_ftpd_use_nfs.py:29
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a NFS "
"filesystem.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил доступ $ACCESS демона ftp к файлам в файловой системе "
"NFS.\n"
"    "

#: ../src/allow_ftpd_use_nfs.py:33
msgid ""
"\n"
"    SELinux prevented the ftp daemon from $ACCESS files stored on a NFS "
"filesystem.\n"
"    NFS (Network Filesystem) is a network filesystem commonly used on Unix / "
"Linux\n"
"    systems.\n"
"\n"
"    The ftp daemon attempted to read one or more files or directories from\n"
"    a mounted filesystem of this type.  As NFS filesystems do not support\n"
"    fine-grained SELinux labeling, all files and directories in the\n"
"    filesystem will have the same security context.\n"
"\n"
"    If you have not configured the ftp daemon to read files from a NFS "
"filesystem\n"
"    this access attempt could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил доступ $ACCESS к файлам на разделе NFS.\n"
"    NFS (Network Filesystem) — сетевая файловая система, обычно используемая "
"в системах UNIX/Linux.\n"
"\n"
"    Демон ftp попытался прочесть данные из смонтированной системы этого типа."
"\n"
"    Так как NFS система не поддерживает низкоуровневое маркирование SELinux, "
"\n"
"    все файлы и каталоги имеют один и тот же контекст безопасности.\n"
"    \n"
"    Если вы не настраивали демон ftp на чтение файлов из NFS,\n"
"    попытка получения доступа может служить сигналом вторжения.\n"
"      "

#: ../src/allow_ftpd_use_nfs.py:47
msgid ""
"\n"
"    Changing the \"allow_ftpd_use_nfs\" boolean to true will allow this "
"access:\n"
"    \"setsebool -P allow_ftpd_use_nfs=1.\"\n"
"    "
msgstr ""
"\n"
"   Установка логического значения «allow_ftpd_use_nfs» разрешит доступ:\n"
"    «setsebool -P allow_ftpd_use_nfs=1»\n"
"    "

#: ../src/allow_ftpd_use_nfs.py:54
msgid ""
" Changing the \"allow_ftpd_use_nfs\" and\n"
"    \"$WRITE_BOOLEAN\" booleans to true will allow this access:\n"
"    \"setsebool -P allow_ftpd_use_nfs=1 $WRITE_BOOLEAN=1\".\n"
"    warning: setting the \"$WRITE_BOOLEAN\" boolean to true will\n"
"    allow the ftp daemon to write to all public content (files and\n"
"    directories with type public_content_t) in addition to writing to\n"
"    files and directories on NFS filesystems.  "
msgstr ""
"Установка логических значений «allow_ftpd_use_nfs» и\n"
"    «$WRITE_BOOLEAN» разрешит такой доступ:\n"
"    «setsebool -P allow_ftpd_use_nfs=1 $WRITE_BOOLEAN=1».\n"
"    Предупреждение: установка значения «$WRITE_BOOLEAN»\n"
"    разрешает демону ftp осуществлять запись во все общие ресурсы\n"
"    (файлы и каталоги типа public_content_t) помимо записи в\n"
"    файлы и каталоги в файловых системах NFS."

#: ../src/allow_ftpd_use_nfs.py:64
msgid "If you want to allow ftpd to write to nfs file systems"
msgstr ""

#: ../src/associate.py:32 ../src/openvpn.py:31 ../src/public_content.py:27
#: ../src/qemu_blk_image.py:27 ../src/qemu_file_image.py:27
#: ../src/restorecon.py:51 ../src/restorecon_source.py:33
#: ../src/rsync_data.py:27 ../src/sshd_root.py:32
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" access to $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает $SOURCE_PATH доступ «$ACCESS» к $TARGET_PATH. \n"
"    "

#: ../src/associate.py:37
#, python-format
msgid ""
"\n"
"You tried to place a type on a %s that is not a file type.  This is not "
"allowed, you must assigne a file type.  You can list all file types using "
"the seinfo command.\n"
"\n"
"seinfo -afile_type -x\n"
"\n"
"    "
msgstr ""
"\n"
"Необходимо выбрать файловый тип для %s. Полный список типов файлов можно "
"получить при помощи команды seinfo.\n"
"\n"
"seinfo -afile_type -x\n"
"\n"
"\n"
"    "

#: ../src/associate.py:44
#, python-format
msgid ""
"If you want to change the label of $TARGET_PATH to %s, you are not allowed "
"to since it is not a valid file type."
msgstr ""

#: ../src/associate.py:49
msgid "you must pick a valid file label."
msgstr "необходимо выбрать действительную метку файла."

#: ../src/automount_exec_config.py:27
msgid ""
"\n"
"    SELinux is preventing the $SOURCE_PATH from executing potentially "
"mislabeled files $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает $SOURCE_PATH использовать потенциально неверно "
"размеченные файлы $TARGET_PATH.\n"
"    "

#: ../src/automount_exec_config.py:31
msgid ""
"\n"
"    SELinux has denied the $SOURCE_PATH from executing potentially\n"
"    mislabeled files $TARGET_PATH.  Automounter can be setup to execute\n"
"    configuration files. If $TARGET_PATH is an automount executable\n"
"    configuration file it needs to have a file label of bin_t.\n"
"    If automounter is trying to execute something that it is not supposed "
"to, this could indicate an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил $SOURCE исполнять потенциально неверно отмеченные\n"
"    файлы $TARGET_PATH. Автомонтировщик может быть настроен на\n"
"    исполнение конфигурационных файлов. Если $TARGET_PATH является\n"
"    исполняемым конфигурационным файлом automount, он должен иметь метку "
"bin_t.\n"
"    Если автомонтировщик пытается выполнить что-то, что он не должен "
"выполнять,\n"
"    это может служить индикатором попытки вторжения.\n"
"    "

#: ../src/automount_exec_config.py:39
msgid ""
"\n"
"    If you want to change the file context of $TARGET_PATH so that the "
"automounter can execute it you can execute \"chcon -t bin_t $TARGET_PATH\".  "
"If you want this to survive a relabel, you need to permanently change the "
"file context: execute  \"semanage fcontext -a -t bin_t '$FIX_TARGET_PATH'\".\n"
"    "
msgstr ""
"\n"
"    Если вы хотите изменить контекст файла $TARGET_PATH, чтобы разрешить его "
"исполнение автомонтировщиком, используйте команду «chcon -t bin_t "
"$TARGET_PATH». Также потребуется изменить контекст файлов, чтобы сохранить "
"его при переразметке: «semanage fcontext -a -t bin_t $FIX_TARGET_PATH»\n"
"    "

#: ../src/bind_ports.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from binding to port $PORT_NUMBER.\n"
"    "
msgstr ""
"\n"
"    SELinux предотвращает сопоставление $SOURCE_PATH порту $PORT_NUMBER.\n"
"    "

#: ../src/bind_ports.py:31
#, python-format
msgid ""
"\n"
"    SELinux has denied the $SOURCE from binding to a network port "
"$PORT_NUMBER which does not have an SELinux type associated with it.\n"
"    If $SOURCE should be allowed to listen on $PORT_NUMBER, use the "
"<i>semanage</i> command to assign $PORT_NUMBER to a port type that "
"$SOURCE_TYPE can bind to (%s).\n"
"    \n"
"\n"
"If $SOURCE is not supposed\n"
"    to bind to $PORT_NUMBER, this could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил $SOURCE подключаться сетевому порту $PORT_NUMBER, "
"которому не присвоен тип SELinux.\n"
"    Если $SOURCE должен иметь возможность подключения к этому порту, можно "
"использовать команду <i>semanage</i> для добавления  $PORT_NUMBER к типу, "
"которому $SOURCE_TYPE может быть сопоставлен (%s).\n"
"    \n"
"Если $SOURCE не должен\n"
"    соответствовать $PORT_NUMBER, возможно, имеет место попытка вторжения.\n"
"    "

#: ../src/bind_ports.py:38
#, python-format
msgid ""
"\n"
"    If you want to allow $SOURCE to bind to port $PORT_NUMBER, you can "
"execute \n"
"\n"
"    # semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"    \n"
"where PORT_TYPE is one of the following: %s.\n"
"\n"
"    \n"
"\n"
"If this system is running as an NIS Client, turning on the allow_ypbind "
"boolean may fix the problem.  setsebool -P allow_ypbind=1.\n"
"    "
msgstr ""
"\n"
"Если вы хотите разрешить сопоставление $SOURCE порту $PORT_NUMBER, выполните\n"
"    # semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"    \n"
"где PORT_TYPE может принимать значения: %s.\n"
"\n"
"    \n"
"\n"
"Если система выполняет роль клиента NIS, можно попробовать установить "
"логическое значение allow_ypbind:\n"
"     «setsebool -P allow_ypbind=1».\n"
"  "

#: ../src/bind_ports.py:47
msgid "If you want to allow $SOURCE_PATH to bind to network port $PORT_NUMBER"
msgstr ""

#: ../src/bind_ports.py:53 ../src/connect_ports.py:51
#, python-format
msgid ""
"# semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"    where PORT_TYPE is one of the following: %s."
msgstr ""
"# semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"    где PORT_TYPE может принимать значения: %s."

#: ../src/bind_ports.py:56 ../src/connect_ports.py:54
#, python-format
msgid "# semanage port -a -t %s -p %s $PORT_NUMBER"
msgstr "# semanage port -a -t %s -p %s $ПОРТ"

#: ../src/catchall_boolean.py:32
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" access on $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает $SOURCE_PATH доступ «$ACCESS» к устройству "
"$TARGET_PATH.\n"
"    "

#: ../src/catchall_boolean.py:36
msgid ""
"\n"
"\n"
"    SELinux denied access requested by $SOURCE. The current boolean\n"
"    settings do not allow this access.  If you have not setup $SOURCE to\n"
"    require this access this may signal an intrusion attempt. If you do "
"intend\n"
"    this access you need to change the booleans on this system to allow\n"
"    the access.\n"
"    "
msgstr ""
"\n"
"\n"
"    SELinux отклонил запрос доступа от $SOURCE. Текущие настройки  \n"
"    логических переменных не разрешают доступ. Такой запрос не \n"
"    ожидается от $SOURCE и может свидетельствовать о \n"
"    попытке вторжения. Если же вы действительно хотите разрешить доступ,  \n"
"    потребуется изменить соответствующие логические переменные. \n"
"    "

#: ../src/catchall_boolean.py:45
msgid ""
"\n"
"    Confined processes can be configured to run requiring different access, "
"SELinux provides booleans to allow you to turn on/off\n"
"    access as needed.\n"
"\n"
"    "
msgstr ""
"\n"
"   Ограничиваемые процессы можно настроить так, чтобы они выполнялись с "
"различными уровнями доступа. При этом доступ можно разрешать и запрещать с "
"помощью логических переменных SELinux.\n"
"\n"
"    "

#: ../src/catchall_boolean.py:62
#, python-format
msgid "If you want to %s"
msgstr ""

#: ../src/catchall_boolean.py:65
#, python-format
msgid "setsebool -P %s %s"
msgstr "setsebool -P %s %s"

#: ../src/catchall_boolean.py:68
#, python-format
msgid "You must tell SELinux about this by enabling the '%s' boolean.\n"
msgstr "Вы должны сообщить SELinux об этом, включив переключатель «%s».\n"

#: ../src/catchall_boolean.py:71
#, python-format
msgid "You can read '%s' man page for more details."
msgstr "Дополнительная документация на '%s' ман странице."

#: ../src/catchall_labels.py:28
#, python-format
msgid ""
"\n"
"    SELinux has denied the $SOURCE access to potentially\n"
"    mislabeled files $TARGET_PATH.  This means that SELinux will not\n"
"    allow httpd to use these files. If httpd should be allowed this access "
"to these files you should change the file context to one of the following "
"types, %s.\n"
"    Many third party apps install html files\n"
"    in directories that SELinux policy cannot predict.  These directories\n"
"    have to be labeled with a file context which httpd can access.\n"
"    "
msgstr ""
"\n"
"    SELinux закрыл $SOURCE доступ к потенциально неверно отмеченным файлам\n"
"    $TARGET_PATH.  Это значит, что SELinux не позволит\n"
"    httpd использовать эти файлы. Чтобы разрешить доступ httpd к этим "
"файлам,\n"
"    измените контекст на один из следующих типов: %s.\n"
"    Многие приложения сторонних производителей устанавливают\n"
"    файлы html в каталоги, которые политика SELinux не может предугадать.\n"
"    Этим каталогам надо присвоить контекст, разрешающий доступ httpd.\n"
"    "

#: ../src/catchall_labels.py:37
msgid "You need to change the label on $FIX_TARGET_PATH"
msgstr "Необходимо изменить метку на $FIX_TARGET_PATH"

#: ../src/catchall_labels.py:40
#, python-format
msgid ""
"# semanage fcontext -a -t FILE_TYPE '$FIX_TARGET_PATH'\n"
"where FILE_TYPE is one of the following: %s.\n"
"Then execute:\n"
"restorecon -v '$FIX_TARGET_PATH'\n"
msgstr ""
"# semanage fcontext -a -t FILE_TYPE '$FIX_TARGET_PATH'\n"
"где FILE_TYPE может принимать значения: %s. \n"
"Затем выполните: \n"
"restorecon -v '$FIX_TARGET_PATH'\n"

#: ../src/catchall.py:29 ../src/chrome.py:29 ../src/mozplugger.py:29
#: ../src/mozplugger_remove.py:29
msgid "SELinux is preventing $SOURCE_PATH \"$ACCESS\" access."
msgstr "SELinux запрещает доступ \"$ACCESS\" для $SOURCE_PATH"

#: ../src/catchall.py:31
msgid ""
"\n"
"\n"
"    SELinux denied access requested by $SOURCE. It is not\n"
"    expected that this access is required by $SOURCE and this access\n"
"    may signal an intrusion attempt. It is also possible that the specific\n"
"    version or configuration of the application is causing it to require\n"
"    additional access.\n"
"\n"
"    "
msgstr ""
"\n"
"\n"
"    SELinux отклонил запрос доступа от $SOURCE. Такой запрос не\n"
"    ожидается от $SOURCE и может свидетельствовать о\n"
"    попытке вторжения. Но не исключено, что отдельные\n"
"    версии или конфигурации приложения потребуют дополнительный\n"
"    доступ.\n"
"\n"
"    "

#: ../src/catchall.py:41
msgid ""
"\n"
"    You can generate a local policy module to allow this\n"
"    access - see <a href=\"http://docs.fedoraproject.org/selinux-faq-fc5/"
"#id2961385\">FAQ</a>\n"
"\n"
"    Please file a bug report.\n"
"    "
msgstr ""
"\n"
"    Чтобы разрешить доступ, можно сгенерировать\n"
"    модуль локальной политики\n"
"    (см. <a href=\"http://docs.fedoraproject.org/selinux-faq-fc5/"
"#id2961385\">FAQ</a>).\n"
"\n"
"    Пожалуйста, создайте отчёт об ошибке.\n"
"    "

#: ../src/catchall.py:50
msgid ""
"If you believe that $SOURCE_BASE_PATH should be allowed $ACCESS access on "
"processes labeled $TARGET_TYPE by default."
msgstr ""

#: ../src/catchall.py:52
msgid ""
"If you believe that $SOURCE_BASE_PATH should have the $ACCESS capability by "
"default."
msgstr ""

#: ../src/catchall.py:53
msgid ""
"If you believe that $SOURCE_BASE_PATH should be allowed $ACCESS access on "
"the $TARGET_BASE_PATH $TARGET_CLASS by default."
msgstr ""

#: ../src/catchall.py:55
msgid ""
"You should report this as a bug.\n"
"You can generate a local policy module to allow this access."
msgstr ""
"Рекомендуется создать отчет об ошибке.\n"
"Чтобы разрешить доступ, можно создать локальный модуль политики."

#: ../src/catchall.py:56
msgid ""
"Allow this access for now by executing:\n"
"# ausearch -c '$SOURCE' --raw | audit2allow -M my-$MODULE_NAME\n"
"# semodule -i my-$MODULE_NAME.pp"
msgstr ""

#: ../src/connect_ports.py:27 ../src/sandbox_connect.py:29
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from connecting to port $PORT_NUMBER.\n"
"    "
msgstr ""
"\n"
"    SELinux предотвращает подключение $SOURCE_PATH к порту $PORT_NUMBER.\n"
"    "

#: ../src/connect_ports.py:31
#, python-format
msgid ""
"\n"
"    SELinux has denied $SOURCE from connecting to a network port "
"$PORT_NUMBER which does not have an SELinux type associated with it.\n"
"    If $SOURCE should be allowed to connect on $PORT_NUMBER, use the "
"<i>semanage</i> command to assign $PORT_NUMBER to a port type that "
"$SOURCE_TYPE can connect to (%s).\n"
"    \n"
"\n"
"If $SOURCE is not supposed\n"
"    to connect to $PORT_NUMBER, this could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/connect_ports.py:38
#, python-format
msgid ""
"\n"
"    If you want to allow $SOURCE to connect to $PORT_NUMBER, you can execute "
"\n"
"\n"
"    # semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"\n"
"    where PORT_TYPE is one of the following: %s.\n"
"    "
msgstr ""
"\n"
"    Чтобы разрешить подключение $SOURCE к порту $PORT_NUMBER, выполните\n"
"\n"
"    # semanage port -a -t PORT_TYPE -p %s $PORT_NUMBER\n"
"\n"
"    где PORT_TYPE может принимать значения: %s.\n"
"  "

#: ../src/connect_ports.py:45 ../src/sandbox_connect.py:47
msgid ""
"If you want to allow $SOURCE_PATH to connect to network port $PORT_NUMBER"
msgstr ""

#: ../src/chrome.py:31
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. It is not expected that this "
"access is required by $SOURCE and this access may signal an intrusion "
"attempt. It is also possible that the specific version or configuration of "
"the application is causing it to require additional access.  spice-xpi run "
"applications within mozilla-plugins that require access to the desktop, that "
"the mozilla_plugin lockdown will not allow, so either you need to turn off "
"the mozilla_plugin lockdown or not use these packages.\n"
"    "
msgstr ""
"\n"
"SELinux отклонил запрос доступа от $SOURCE. Такой запрос от $SOURCE не "
"ожидается и может сигнализировать о\n"
"попытке вторжения. Возможно также, что отдельные версии или настройки "
"приложения пытаются получить дополнительный доступ. spice-xpi запускает "
"приложения в mozilla-plugins, что требует доступа к рабочему столу, но "
"запрещается блокировкой mozilla_plugin. В этом случае надо либо отключить "
"блокировку mozilla_plugin, либо отказаться от использования таких пакетов.\n"
"   "

#: ../src/chrome.py:35
msgid ""
"\n"
"Either remove the mozplugger or spice-xpi package by executing 'yum remove "
"mozplugger spice-xpi', or turn off enforcement of SELinux over the Chrome "
"plugins. setsebool -P unconfined_chrome_sandbox_transition 0\n"
"    "
msgstr ""
"\n"
"Удалите пакет mozplugger или spice-xpi, выполнив «yum remove mozplugger "
"spice-xpi», или отключите контроль SELinux для плагинов Chrome. setsebool -P "
"unconfined_chrome_sandbox_transition 0\n"
"    "

#: ../src/chrome.py:38 ../src/mozplugger.py:38
#, python-format
msgid "If you want to use the %s package"
msgstr ""

#: ../src/chrome.py:43
msgid "you must turn off SELinux controls on the Chrome plugins."
msgstr "нужно отключить управление SELinux для плагинов Chrome."

#: ../src/cvs_data.py:27
msgid ""
"\n"
"    SELinux is preventing cvs ($SOURCE_PATH) \"$ACCESS\" access to "
"$TARGET_PATH\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает cvs ($SOURCE_PATH) доступ «$ACCESS» к $TARGET_PATH. \n"
"    "

#: ../src/cvs_data.py:31
msgid ""
"\n"
"    SELinux denied cvs access to $TARGET_PATH.\n"
"    If this is a CVS repository it needs to have a file context label of\n"
"    cvs_data_t. If you did not intend to use $TARGET_PATH as a CVS "
"repository\n"
"    it could indicate either a bug or it could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил доступ cvs к $TARGET_PATH.\n"
"    Если это репозиторий CVS, он должен иметь контекст\n"
"    cvs_data_t. Если вы не используете $TARGET_PATH как репозиторий CVS,\n"
"    возможно, это ошибка или попытка вторжения.\n"
"    "

#: ../src/cvs_data.py:38
msgid ""
"\n"
"    You can alter the file context by executing chcon -R -t cvs_data_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"cvs_data_t '$FIX_TARGET_PATH'\"\n"
"    \n"
"    "
msgstr ""
"\n"
"    Можно изменить контекст, выполнив команду: \n"
"    «chcon -R -t cvs_data_t '$TARGET_PATH'»\n"
"    Также необходимо изменить контексты файлов в системе, \n"
"    используемые по умолчанию, \n"
"    чтобы они сохранились даже при повторной разметке. \n"
"    «semanage fcontext -a -t cvs_data_t '$FIX_TARGET_PATH'»\n"
"    \n"
"    "

#: ../src/cvs_data.py:44
msgid "If $TARGET_BASE_PATH should be shared via the cvs daemon"
msgstr ""

#: ../src/cvs_data.py:45 ../src/qemu_file_image.py:49
msgid "You need to change the label on $TARGET_BASE_PATH'"
msgstr "Необходимо изменить метку для $TARGET_BASE_PATH'"

#: ../src/dac_override.py:28
msgid "\n"
"    SELinux is preventing $SOURCE_PATH the \"$ACCESS\" capability.\n"
"    "
msgstr "\n"
"    SELinux запретил операцию \"$ACCESS\" для $SOURCE_PATH.  \n"
"    "

#: ../src/dac_override.py:32
msgid ""
"\n"
"\tdac_override and dac_read_search capabilities usually indicates that the "
"root process does not have access to a file based on the permission flags.  "
"This usually mean you have some file with the wrong ownership/permissions on "
"it.\n"
"    "
msgstr ""
"\n"
"\tdac_override и dac_read_search служат знаком того, что у корневого "
"процесса нет доступа к файлу, разрешения которого определяются флагами. "
"Обычно это характеризует файл с неверно назначенными разрешениями.\n"
"    "

#: ../src/dac_override.py:38
msgid ""
"If you want to help identify if domain needs this access or you have a file "
"with the wrong permissions on your system"
msgstr ""

#: ../src/dac_override.py:39
msgid ""
"turn on full auditing to get path information about the offending file and "
"generate the error again."
msgstr ""
"включите полный аудит, чтобы определить путь к конфликтному файлу и повторно "
"сгенерировать ошибку."

#: ../src/dac_override.py:40
msgid ""
"\n"
"Turn on full auditing\n"
"# auditctl -w /etc/shadow -p w\n"
"Try to recreate AVC. Then execute\n"
"# ausearch -m avc -ts recent\n"
"If you see PATH record check ownership/permissions on file, and fix it,\n"
"otherwise report as a bugzilla."
msgstr ""
"\n"
"Включите аудит:\n"
"# auditctl -w /etc/shadow -p w\n"
"Попытайтесь заново создать AVC, после чего выполните:\n"
"# ausearch -m avc -ts recent\n"
"Если запись PATH осуществляет проверку разрешений файла, исправьте это,  \n"
"в противном случае создайте запрос в Bugzilla."

#: ../src/device.py:29
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" access to device "
"$TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает $SOURCE_PATH доступ «$ACCESS» к устройству "
"$TARGET_PATH.  \n"
"    "

#: ../src/device.py:33
msgid ""
"\n"
"\n"
"    SELinux has denied $SOURCE \"$ACCESS\" access to device $TARGET_PATH.\n"
"    $TARGET_PATH is mislabeled, this device has the default label of the /"
"dev directory, which should not\n"
"    happen.  All Character and/or Block Devices should have a label.\n"
"\n"
"    You can attempt to change the label of the file using\n"
"\n"
"    restorecon -v '$TARGET_PATH'.\n"
"\n"
"    If this device remains labeled device_t, then this is a bug in SELinux "
"policy.\n"
"\n"
"    Please file a bg report.\n"
"\n"
"    If you look at the other similar devices labels, ls -lZ /dev/SIMILAR, "
"and find a type that would work for $TARGET_PATH,\n"
"    you can use chcon -t SIMILAR_TYPE '$TARGET_PATH', If this fixes the "
"problem, you can make this permanent by executing\n"
"    semanage fcontext -a -t SIMILAR_TYPE '$FIX_TARGET_PATH'\n"
"\n"
"    If the restorecon changes the context, this indicates that the "
"application that created the device, created it without\n"
"    using SELinux APIs.  If you can figure out which application created the "
"device, please file a bug report against this application.\n"
"\n"
"    "
msgstr ""
"\n"
"\n"
"    SELinux запретил $SOURCE доступ «$ACCESS» к устройству $TARGET_PATH.\n"
"    $TARGET_PATH отмечено неверно и имеет стандартную метку каталога /dev.\n"
"    Все символьные и блочные устройства должны иметь свою метку.\n"
"\n"
"    Можно попробовать изменить метку, выполнив команду\n"
"\n"
"    restorecon -v '$TARGET_PATH'.\n"
"\n"
"    Если устройство по прежнему отмечено как device_t, это означает ошибку в "
"политике SELinux.\n"
"\n"
"    Тогда, пожалуйста, создайте запрос в Bugzilla.\n"
"\n"
"    Если вы просмотрите метки схожих устройств с помощью команды \n"
"    ls -lZ /dev/SIMILAR и найдете тип, который подходит $TARGET_PATH,\n"
"    можно выполнить chcon -t SIMILAR_TYPE '$TARGET_PATH'.\n"
"    Если это поможет, сохраните изменения:\n"
"    semanage fcontext -a -t SIMILAR_TYPE '$TARGET_PATH'\n"
"\n"
"    Если restorecon изменяет контекст, это означает, что приложение, "
"создавшее устройство, не использовало\n"
"    SELinux API.  Если вы знаете, какое приложение создало это устройство, "
"пожалуйста, создайте запрос в Bugzilla для этого приложения.\n"
"\n"
"    "

#: ../src/device.py:56
msgid ""
"\n"
"    Attempt restorecon -v '$TARGET_PATH' or chcon -t SIMILAR_TYPE "
"'$TARGET_PATH'\n"
"    "
msgstr ""
"\n"
"    Попробуйте выполнить «restorecon -v $TARGET_PATH» или «chcon -t "
"SIMILAR_TYPE '$TARGET_PATH'»\n"
"    "

#: ../src/device.py:60
msgid ""
"You need to change the label on $TARGET_PATH to a type of a similar device."
msgstr ""
"Необходимо изменить метку для $FIX_TARGET_PATH на тип аналогичного "
"устройства."

#: ../src/device.py:61
msgid ""
"# semanage fcontext -a -t SIMILAR_TYPE '$FIX_TARGET_PATH'\n"
"# restorecon -v '$FIX_TARGET_PATH'"
msgstr ""
"# semanage fcontext -a -t SIMILAR_TYPE '$FIX_TARGET_PATH'\n"
"# restorecon -v '$FIX_TARGET_PATH'"

#: ../src/disable_ipv6.py:30
msgid "\n"
"Disable IPV6 properly.\n"
"    "
msgstr "\n"
"Отключите IPV6.\n"
"    "

#: ../src/disable_ipv6.py:40
msgid "If you want to disable IPV6 on this machine"
msgstr ""

#: ../src/disable_ipv6.py:41
msgid ""
"you need to set /proc/sys/net/ipv6/conf/all/disable_ipv6 to 1 and do not "
"blacklist the module'"
msgstr ""
"измените значение /proc/sys/net/ipv6/conf/all/disable_ipv6 на 1 и не вносите "
"модуль в черный список"

#: ../src/disable_ipv6.py:42
msgid "Add \n"
"net.ipv6.conf.all.disable_ipv6 = 1\n"
"to /etc/sysctl.conf\n"
msgstr "Добавьте \n"
"net.ipv6.conf.all.disable_ipv6 = 1\n"
"в /etc/sysctl.conf\n"

#: ../src/file.py:27
msgid ""
"\n"
"    SELinux is preventing access to files with the label, file_t.\n"
"    "
msgstr "\n"
"    SELinux предотвращает доступ к файлам с меткой file_t.\n"
"    "

#: ../src/file.py:31
msgid ""
"\n"
"    SELinux permission checks on files labeled file_t are being\n"
"    denied.  file_t is the context the SELinux kernel gives to files\n"
"    that do not have a label. This indicates a serious labeling\n"
"    problem. No files on an SELinux box should ever be labeled file_t.\n"
"    If you have just added a disk drive to the system you can\n"
"    relabel it using the restorecon command.  For example if you saved the\n"
"home directory from a previous installation that did not use SELinux, "
"'restorecon -R -v /home' will fix the labels.  Otherwise you should\n"
"    relabel the entire file system.\n"
"    "
msgstr ""
"\n"
"Файлы с меткой file_t не могут пройти проверку SELinux.\n"
"Контекст file_t назначается ядром SELinux всем файлам без меток.\n"
"Это служит знаком серьезного нарушения порядка разметки, \n"
"так как файлы не должны отмечены как file_t.\n"
"Если вы добавили новый диск, ему можно присвоить метку при помощи restorecon."
" Если вы сохранили домашний каталог из предыдущей установки, которая не "
"использовала SELinux, \n"
"команда «restorecon -R -v /home» поможет исправить метки.\n"
"В противном случае придется повторно разметить всю файловую систему.\n"
"    "

#: ../src/file.py:42
msgid ""
"\n"
"    You can execute the following command as root to relabel your\n"
"    computer system: \"touch /.autorelabel; reboot\"\n"
"    "
msgstr ""
"\n"
"    Выполните следующую команду в режиме root\n"
"    для изменения маркировки системы: «touch /.autorelabel; reboot»\n"
"    "

#: ../src/file.py:49
msgid "This is caused by a newly created file system."
msgstr ""

#: ../src/file.py:51
msgid "If you think this is caused by a badly mislabeled machine."
msgstr ""

#: ../src/file.py:55
msgid "you need to add labels to it."
msgstr "необходимо добавить метки."

#: ../src/file.py:57
msgid "you need to fully relabel."
msgstr "необходимо заново осуществить переразметку."

#: ../src/filesystem_associate.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH from creating a file with a context "
"of $SOURCE_TYPE on a filesystem.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает $SOURCE_PATH создавать в файловой системе файл с "
"контекстом $SOURCE_TYPE.\n"
"    "

#: ../src/filesystem_associate.py:31
msgid ""
"\n"
"    SELinux is preventing $SOURCE from creating a file with a context of "
"$SOURCE_TYPE on a filesystem.\n"
"    Usually this happens when you ask the cp command to maintain the context "
"of a file when\n"
"    copying between file systems, \"cp -a\" for example.  Not all file "
"contexts should be maintained\n"
"    between the file systems.  For example, a read-only file type like "
"iso9660_t should not be placed\n"
"    on a r/w system.  \"cp -p\" might be a better solution, as this will "
"adopt the default file context\n"
"    for the destination.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил $SOURCE создавать файл с контекстом $SOURCE_TYPE в "
"файловой системе.\n"
"    Обычно это происходит при использовании команды «cp» для сохранения "
"контекста файла при\n"
"    копировании (например, «cp -a»). Не все файловые контексты должны "
"сохранять постоянство в разных файловых системах.\n"
"    Например, тип iso9660_t доступных только для чтения файлов не должен "
"использоваться в\n"
"    системах, допускающих запись. В этом случае лучше подойдёт команда «cp -"
"p», так как она адаптирует\n"
"    файловый контекст в зависимости от назначения.\n"
"    "

#: ../src/filesystem_associate.py:40
msgid ""
"\n"
"    Use a command like \"cp -p\" to preserve all permissions except SELinux "
"context.\n"
"    "
msgstr ""
"\n"
"    Используйте команду наподобие «cp -p» для сохранения постоянства всех "
"разрешений кроме контекста SELinux.\n"
"    "

#: ../src/filesystem_associate.py:43
msgid ""
"If you believe $SOURCE_BASE_PATH should be allowed to create "
"$TARGET_BASE_PATH files"
msgstr ""

#: ../src/filesystem_associate.py:44
msgid ""
"you need to use a different command. You are not allowed to preserve the "
"SELinux context on the target file system."
msgstr ""
"необходимо выбрать другую команду. Контекст SELinux целевой файловой системы "
"не может быть сохранен."

#: ../src/filesystem_associate.py:46
msgid ""
"use a command like \"cp -p\" to preserve all permissions except SELinux "
"context."
msgstr ""
"используйте команду наподобие «cp -p» для сохранения постоянства всех "
"разрешений кроме контекста SELinux."

#: ../src/httpd_can_sendmail.py:28
msgid "\n"
"    SELinux is preventing the http daemon from sending mail.\n"
"    "
msgstr "\n"
"   SELinux предотвращает отправление почты демоном http.\n"
"     "

#: ../src/httpd_can_sendmail.py:32
msgid ""
"\n"
"    SELinux has denied the http daemon from sending mail. An\n"
"    httpd script is trying to connect to a mail port or execute the\n"
"    sendmail command. If you did not setup httpd to sendmail, this could\n"
"    signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/httpd_can_sendmail.py:39
msgid ""
"\n"
"    If you want httpd to send mail you need to turn on the\n"
"    $BOOLEAN boolean: \"setsebool -P\n"
"    $BOOLEAN=1\"\n"
"    "
msgstr ""
"\n"
"    Для разрешения отправки почты демоном httpd\n"
"    установите значение $BOOLEAN:\n"
"    «setsebool -P $BOOLEAN=1»\n"
"    "

#: ../src/httpd_can_sendmail.py:47
msgid "If you want to allow httpd to send mail"
msgstr ""

#: ../src/httpd_can_sendmail.py:48
msgid "you must setup SELinux to allow this"
msgstr "необходимо разрешить SELinux выполнять эти действия."

#: ../src/httpd_unified.py:29
msgid "\n"
"    SELinux prevented httpd $ACCESS access to http files.\n"
"    "
msgstr "\n"
"    SELinux запретил доступ httpd ($ACCESS) к файлам http.\n"
"    "

#: ../src/httpd_unified.py:33
msgid ""
"\n"
"    SELinux prevented httpd $ACCESS access to http files.\n"
"\n"
"    Ordinarily httpd is allowed full access to all files labeled with http "
"file\n"
"    context.  This machine has a tightened security policy with the $BOOLEAN\n"
"    turned off,  This requires explicit labeling of all files.  If a file is\n"
"    a cgi script it needs to be labeled with httpd_TYPE_script_exec_t in "
"order\n"
"    to be executed.  If it is read only content, it needs to be labeled\n"
"    httpd_TYPE_content_t, it is writable content. it needs to be labeled\n"
"    httpd_TYPE_script_rw_t or httpd_TYPE_script_ra_t. You can use the\n"
"    chcon command to change these context.  Please refer to the man page\n"
"    \"man httpd_selinux\" or\n"
"    <a href=\"http://fedora.redhat.com/docs/selinux-apache-fc3\">FAQ</a>\n"
"    \"TYPE\" refers to one of \"sys\", \"user\" or \"staff\" or potentially "
"other\n"
"    script types.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил доступ $ACCESS для httpd к файлам http.\n"
"\n"
"    Обычно httpd обладает полным доступом к файлам\n"
"    с контекстом http.  Уровень защиты этого компьютера усилен\n"
"    в то время как переменная $BOOLEAN отключена,\n"
"    поэтому необходимо отдельно присвоить метки файлам.\n"
"    Если файл представляет собой сценарий cgi,\n"
"    ему должна быть присвоена метка httpd_TYPE_script_exec_t.\n"
"    Если файл предназначен только для чтения, \n"
"    он должен быть отмечен как  httpd_TYPE_content_t.\n"
"    Если же файл должен быть доступен для записи,     он должен быть отмечен "
"как httpd_TYPE_script_rw_t или \n"
"    httpd_TYPE_script_ra_t. Команда chcon позволяет изменить контекст.\n"
"    За подробной информацией обратитесь к справочной странице \n"
"    \"man httpd_selinux\" или  \n"
"    <a href=\"http://fedora.redhat.com/docs/selinux-apache-fc3\">FAQ</a>.\n"
"    «TYPE» может принимать значения «sys», «user», «staff» и \n"
"    некоторые другие значения типов сценариев.\n"
"    "

#: ../src/httpd_unified.py:50
msgid ""
"\n"
"    Changing the \"$BOOLEAN\" boolean to true will allow this access:\n"
"    \"setsebool -P $BOOLEAN=1\"\n"
"    "
msgstr ""
"\n"
"    Установка логического значения «$BOOLEAN» в TRUE разрешит доступ:\n"
"    «setsebool -P $BOOLEAN=1»\n"
"    "

#: ../src/httpd_unified.py:57
msgid ""
"If you want to allow httpd to execute cgi scripts and to unify HTTPD "
"handling of all content files."
msgstr ""

#: ../src/httpd_unified.py:58
msgid ""
"you must tell SELinux about this by enabling the 'httpd_unified' and "
"'http_enable_cgi' booleans"
msgstr ""
"установите переменные «httpd_unified» и «http_enable_cgi», чтобы сообщить об "
"этом SELinux."

#: ../src/httpd_write_content.py:30
msgid ""
"\n"
"    SELinux prevented httpd $ACCESS access to $TARGET_PATH.\n"
"\n"
"    httpd scripts are not allowed to write to content without explicit\n"
"    labeling of all files.  If $TARGET_PATH is writable content. it needs\n"
"    to be labeled httpd_sys_rw_content_t or if all you need is append you "
"can label it httpd_sys_ra_content_t.   Please refer to 'man httpd_selinux' "
"for more information on setting up httpd and selinux.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает httpd ($SOURCE_PATH) доступ $ACCESS к $TARGET_PATH.\n"
"\n"
"    Сценариям httpd не разрешено осуществлять запись без \n"
"    явной маркировки всех файлов. Если $TARGET_PATH допускает запись, \n"
"    то его надо отметить как httpd_sys_rw_content_t или "
"httpd_sys_ra_content_t\n"
"    (если требуется только добавление).\n"
"    Подробную информацию можно найти на странице помощи\n"
"    «man httpd_selinux».\n"
"    "

#: ../src/httpd_write_content.py:40 ../src/qemu_blk_image.py:45
#: ../src/samba_share.py:45 ../src/swapfile.py:45 ../src/xen_image.py:48
msgid "You need to change the label on '$FIX_TARGET_PATH'"
msgstr "Необходимо изменить метку на «$FIX_TARGET_PATH»"

#: ../src/kernel_modules.py:29
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH tried to modify "
"kernel configuration.\n"
"    "
msgstr ""
"\n"
"    Ваша система может быть серьёзно скомпрометирована! $SOURCE_PATH "
"пытается изменить конфигурацию ядра.\n"
"    "

#: ../src/kernel_modules.py:33
msgid ""
"\n"
"    SELinux has prevented $SOURCE from modifying $TARGET.  This denial\n"
"    indicates $SOURCE was trying to modify the way the kernel runs or to\n"
"    actually insert code into the kernel. All applications that need this\n"
"    access should have already had policy written for them.  If a "
"compromised\n"
"    application tries to modify the kernel this AVC will be generated. This "
"is a\n"
"    serious issue. Your system may very well be compromised.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил $SOURCE изменять $TARGET.  \n"
"    $SOURCE пытался изменить поведение ядра или вставить код в ядро. \n"
"    Для всех приложений, которым необходим такой уровень доступа, \n"
"    должна существовать отдельная политика. \n"
"    Если скомпрометированное приложение попытается модифицировать ядро,\n"
"    будет сгенерировано AVC. \n"
"    Это является серьёзной проблемой и может служить индикатором того, \n"
"    что система была скомпрометирована.\n"
"     "

#: ../src/kernel_modules.py:44 ../src/selinuxpolicy.py:44
msgid ""
"If you do not think $SOURCE_BASE_PATH should try $ACCESS access on "
"$TARGET_BASE_PATH."
msgstr ""

#: ../src/kernel_modules.py:45 ../src/selinuxpolicy.py:45
msgid ""
"you may be under attack by a hacker, since confined applications should not "
"need this access."
msgstr ""
"возможно, вы подверглись хакерской атаке, так как защищенные приложения не "
"должны запрашивать этот уровень доступа."

#: ../src/leaks.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH access to a leaked $TARGET_PATH file "
"descriptor.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает $SOURCE_PATH доступ к дескриптору файла $TARGET_PATH "
"(возможна утечка).\n"
"    "

#: ../src/leaks.py:31
msgid ""
"\n"
"    SELinux denied access requested by the $SOURCE command. It looks like "
"this is either a leaked descriptor or $SOURCE output was redirected to a "
"file it is not allowed to access.  Leaks usually can be ignored since "
"SELinux is just closing the leak and reporting the error.  The application "
"does not use the descriptor, so it will run properly.  If this is a "
"redirection, you will not get output in the $TARGET_PATH.  You should "
"generate a bugzilla on selinux-policy, and it will get routed to the "
"appropriate package.  You can safely ignore this avc.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил запрос доступа, полученный от команды $SOURCE. Вероятна "
"утечка дескриптора или вывод $SOURCE был перенаправлен в файл, к которому у "
"нее нет доступа. Утечки обычно можно игнорировать, так как SELinux просто их "
"закрывает и сообщает об ошибке. Приложение не использует дескриптор, поэтому "
"его работа нарушена не будет. Если же причиной является перенаправление, "
"$TARGET_PATH не будет содержать вывод. В этом случае создайте запрос в "
"Bugzilla для компонента selinux-policy.\n"
"    "

#: ../src/leaks.py:35
msgid ""
"\n"
"    You can generate a local policy module to allow this\n"
"    access - see <a href=\"http://docs.fedoraproject.org/selinux-faq-fc5/"
"#id2961385\">FAQ</a>\n"
"    "
msgstr ""
"\n"
"    Чтобы разрешить доступ, можно сгенерировать \n"
"    модуль локальной политики\n"
"    (см. <a href=\"http://docs.fedoraproject.org/selinux-faq-fc5/"
"#id2961385\">FAQ</a>).\n"
"    "

#: ../src/leaks.py:42
msgid ""
"If you want to ignore $SOURCE_BASE_PATH trying to $ACCESS access the "
"$TARGET_BASE_PATH $TARGET_CLASS, because you believe it should not need this "
"access."
msgstr ""

#: ../src/leaks.py:43
msgid ""
"You should report this as a bug.  \n"
"You can generate a local policy module to dontaudit this access."
msgstr ""
"Рекомендуется создать отчет об ошибке.\n"
"Чтобы отменить аудит подобного доступа, можно создать локальный модуль "
"политики."

#: ../src/leaks.py:44
msgid ""
"# ausearch -x $SOURCE_PATH --raw | audit2allow -D -M my-$SOURCE\n"
"# semodule -i my-$SOURCE.pp"
msgstr ""

#: ../src/mmap_zero.py:29
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH attempted to mmap "
"low kernel memory.\n"
"    "
msgstr ""
"\n"
"    Ваша система может быть серьёзно скомпрометирована!\n"
"    $SOURCE_PATH пытается выполнить mmap для нижнего диапазона памяти ядра.\n"
"    "

#: ../src/mmap_zero.py:33
msgid ""
"\n"
"    SELinux has denied the $SOURCE the ability to mmap low area of the "
"kernel \n"
"    address space.  The ability to mmap a low area of the address space is \n"
"    configured by /proc/sys/kernel/mmap_min_addr.  Preventing such mappings \n"
"    helps protect against exploiting null deref bugs in the kernel. All \n"
"    applications that need this access should have already had policy "
"written \n"
"    for them.  If a compromised application tries to modify the kernel, this "
"AVC \n"
"    would be generated. This is a serious issue. Your system may very well "
"be \n"
"    compromised.\n"
"    "
msgstr ""

#: ../src/mmap_zero.py:46
msgid ""
"If you do not think $SOURCE_PATH should need to mmap low memory in the "
"kernel."
msgstr ""

#: ../src/mmap_zero.py:47
msgid "you may be under attack by a hacker, this is a very dangerous access."
msgstr ""
"возможно, вы подверглись хакерской атаке. Это обращение представляет риск "
"безопасности."

#: ../src/mounton.py:29
msgid ""
"\n"
"    SELinux prevented $SOURCE from mounting on the file or directory\n"
"    \"$TARGET_PATH\" (type \"$TARGET_TYPE\").\n"
"    "
msgstr ""
"\n"
"    SELinux запретил $SOURCE монтировать в\n"
"    файл или каталог «$TARGET_PATH» (тип «$TARGET_TYPE»).\n"
"    "

#: ../src/mounton.py:34
msgid ""
"\n"
"    SELinux prevented $SOURCE from mounting a filesystem on the file\n"
"    or directory \"$TARGET_PATH\" of type \"$TARGET_TYPE\". By default\n"
"    SELinux limits the mounting of filesystems to only some files or\n"
"    directories (those with types that have the mountpoint attribute). The\n"
"    type \"$TARGET_TYPE\" does not have this attribute. You can change the\n"
"    label of the file or directory.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил $SOURCE монтировать файловую систему в\n"
"    «$TARGET_PATH» с типом «$TARGET_TYPE». По умолчанию\n"
"    SELinux ограничивает монтирование файловых систем \n"
"    некоторыми файлами или\n"
"    каталогами (имеющими атрибут mountpoint). \n"
"    Тип «$TARGET_TYPE» не имеет этого атрибута.  \n"
"    Можно попробовать изменить метку файла или каталога.\n"
"    "

#: ../src/mounton.py:43
msgid ""
"\n"
"    Changing the file_context to mnt_t will allow mount to mount the file "
"system:\n"
"    \"chcon -t mnt_t '$TARGET_PATH'.\"\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"mnt_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Замена file_context на mnt_t позволит монтировать файловую систему:\n"
"    «chcon -t mnt_t '$TARGET_PATH'.»\n"
"    Также необходимо изменить контексты файлов в системе, \n"
"    используемые по умолчанию, \n"
"    чтобы они сохранились даже при повторной разметке. \n"
"    «semanage fcontext -a -t mnt_t '$FIX_TARGET_PATH'»\n"
"    "

#: ../src/mounton.py:48
msgid "If you want to allow $SOURCE_BASE_PATH to mount on $TARGET_BASE_PATH."
msgstr ""

#: ../src/mounton.py:49
msgid "you must change the labeling on $TARGET_PATH."
msgstr "необходимо изменить разметку $TARGET_PATH."

#: ../src/mozplugger.py:31
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. It is not expected that this "
"access is required by $SOURCE and this access may signal an intrusion "
"attempt. It is also possible that the specific version or configuration of "
"the application is causing it to require additional access.  mozplugger and "
"spice-xpi run applications within mozilla-plugins that require access to the "
"desktop, that the mozilla_plugin lockdown will not allow, so either you need "
"to turn off the mozilla_plugin lockdown or not use these packages.\n"
"    "
msgstr ""
"\n"
"SELinux отклонил запрос доступа от $SOURCE. Такой запрос от $SOURCE не "
"ожидается и может сигнализировать о\n"
"попытке вторжения. Возможно также, что отдельные версии или настройки "
"приложения пытаются получить дополнительный доступ. mozplugger и spice-xpi "
"запускают приложения в mozilla-plugins, что требует доступа к рабочему "
"столу, но запрещается блокировкой mozilla_plugin. В этом случае надо либо "
"отключить блокировку mozilla_plugin, либо отказаться от использования таких "
"пакетов.\n"
"   "

#: ../src/mozplugger.py:35
msgid ""
"\n"
"Either remove the mozplugger or spice-xpi package by executing 'yum remove "
"mozplugger spice-xpi' or turn off enforcement of SELinux over the Firefox "
"plugins. setsebool -P unconfined_mozilla_plugin_transition 0\n"
"    "
msgstr ""
"\n"
"Удалите пакеты mozplugger и spice-xpi, выполнив «yum remove mozplugger spice-"
"xpi», или отключите контроль SELinux для плагинов Chrome. setsebool -P "
"unconfined_mozilla_plugin_transition 0\n"
"    "

#: ../src/mozplugger.py:43
msgid "you must turn off SELinux controls on the Firefox plugins."
msgstr "нужно отключить управление SELinux для плагинов Firefox."

#: ../src/mozplugger_remove.py:31
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. It is not\n"
"    expected that this access is required by $SOURCE and this access\n"
"    may signal an intrusion attempt. It is also possible that the specific\n"
"    version or configuration of the application is causing it to require\n"
"    additional access.\n"
"    "
msgstr ""
"\n"
"SELinux отклонил запрос доступа от $SOURCE. \n"
"Такой запрос не ожидается от $SOURCE и может свидетельствовать о\n"
"попытке вторжения. Но не исключено, что отдельные версии и \n"
"конфигурации приложения потребуют дополнительный доступ.\n"
"\n"
"    "

#: ../src/mozplugger_remove.py:39
msgid ""
"\n"
"Either remove the mozplluger package by executing 'yum remove mozplugger'\n"
"Or turn off enforcement of SELinux over the Firefox plugins.\n"
"setsebool -P unconfined_mozilla_plugin_transition 0\n"
"    "
msgstr ""
"\n"
"Либо удалите пакет mozplluger командой «yum remove mozplugger»,\n"
"либо отключите применение SELinux к подключаемым модулям Firefox.\n"
"setsebool -P unconfined_mozilla_plugin_transition 0\n"
"    "

#: ../src/mozplugger_remove.py:44
msgid ""
"If you want to to continue using SELinux Firefox plugin containment rather "
"then using mozplugger package"
msgstr ""

#: ../src/mozplugger_remove.py:45
msgid "you must remove the mozplugger package."
msgstr "необходимо удалить пакет mozplugger."

#: ../src/openvpn.py:35
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $TARGET_PATH may\n"
"    be mislabeled. openvpn is allowed to read content in home directory if "
"it\n"
"    is labeled correctly.\n"
"    "
msgstr ""

#: ../src/openvpn.py:41
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.  \n"
"    # restorecon -R /root/.ssh\n"
"    "
msgstr ""

#: ../src/openvpn.py:49
msgid ""
"If you want to mv $TARGET_BASE_PATH to standard location so that "
"$SOURCE_BASE_PATH can have $ACCESS access"
msgstr ""

#: ../src/openvpn.py:51
msgid ""
"If you want to modify the label on $TARGET_BASE_PATH so that "
"$SOURCE_BASE_PATH can have $ACCESS access on it"
msgstr ""

#: ../src/openvpn.py:55
msgid "you must move the cert file to the ~/.cert directory"
msgstr "файл сертификата необходимо переместить в каталог ~/.cert"

#: ../src/openvpn.py:57 ../src/sshd_root.py:48
msgid "you must fix the labels."
msgstr "необходимо исправить метки."

#: ../src/public_content.py:31
msgid ""
"\n"
"    SELinux denied access to $TARGET_PATH requested by $SOURCE.\n"
"    $TARGET_PATH has a context used for sharing by a different program. If "
"you\n"
"    would like to share $TARGET_PATH from $SOURCE also, you need to\n"
"    change its file context to public_content_t.  If you did not intend to\n"
"    allow this access, this could signal an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил доступ к $TARGET_PATH по запросу $SOURCE.\n"
"    $TARGET_PATH имеет контекст, совместно используемый другими приложениями."
" Если\n"
"    вы хотите использовать $TARGET_PATH из $SOURCE, то нужно\n"
"    изменить его контекст на public_content_t.  Если вы не планировали такой "
"вариант, \n"
"    это может служить индикатором попытки вторжения.\n"
"    "

#: ../src/public_content.py:39
msgid ""
"\n"
"    You can alter the file context by executing chcon -t public_content_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"public_content_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Файловый контекст можно изменить, выполнив\n"
"    «chcon -t public_content_t '$TARGET_PATH'»\n"
"    Также необходимо изменить контексты файлов в системе, \n"
"    используемые по умолчанию, \n"
"    чтобы они сохранились даже при повторной разметке. \n"
"    «semanage fcontext -a -t public_content_t '$FIX_TARGET_PATH'»\n"
"    "

#: ../src/public_content.py:50
msgid "If you want to treat $TARGET_BASE_PATH as public content"
msgstr ""

#: ../src/public_content.py:51
msgid ""
"You need to change the label on $TARGET_BASE_PATH to public_content_t or "
"public_content_rw_t."
msgstr ""
"Необходимо изменить метку $TARGET_BASE_PATH с public_content_t на "
"public_content_rw_t."

#: ../src/qemu_blk_image.py:31
msgid ""
"\n"
"    SELinux denied qemu access to the block device $TARGET_PATH.\n"
"    If this is a virtualization image, it needs to be labeled with a "
"virtualization file context (virt_image_t). You can relabel $TARGET_PATH to "
"be virt_image_t using chcon.  You also need to execute semanage fcontext -a -"
"t virt_image_t '$FIX_TARGET_PATH' to add this\n"
"    new path to the system defaults. If you did not intend to use "
"$TARGET_PATH as a qemu\n"
"    image it could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    SELinux закрыл доступ qemu к $TARGET_PATH.\n"
"    Если это образ виртуализации, он должен иметь метку virt_image_t.\n"
"    Можно изменить метку $TARGET_PATH на virt_image_t с помощью chcon.\n"
"    Также выполните команду\n"
"    «semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH'»\n"
"    для добавления нового пути в системный список стандартных путей.\n"
"    Если вы не планировали использовать\n"
"    $TARGET_PATH как образ qemu, возможно, это ошибка или попытка вторжения.\n"
"    "

#: ../src/qemu_blk_image.py:38 ../src/qemu_file_image.py:41
msgid ""
"\n"
"    You can alter the file context by executing chcon -t virt_image_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"virt_image_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Контекст файла можно изменить, выполнив команду\n"
"    «chcon -t virt_image_t '$TARGET_PATH'»\n"
"    Также необходимо изменить контексты файлов в системе, \n"
"    используемые по умолчанию, \n"
"    чтобы они сохранились даже при повторной разметке. \n"
"    «semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH»\n"
"    "

#: ../src/qemu_blk_image.py:46
msgid ""
"# semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH'\n"
"# restorecon -v '$FIX_TARGET_PATH'"
msgstr ""
"# semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH'\n"
"# restorecon -v '$FIX_TARGET_PATH'"

#: ../src/qemu_file_image.py:31
msgid ""
"\n"
"    SELinux denied qemu access to $TARGET_PATH.\n"
"    If this is a virtualization image, it has to have a file context label "
"of\n"
"    virt_image_t. The system is setup to label image files in directory./var/"
"lib/libvirt/images\n"
"    correctly.  We recommend that you copy your image file to /var/lib/"
"libvirt/images.\n"
"    If you really want to have your qemu image files in the current "
"directory, you can relabel $TARGET_PATH to be virt_image_t using chcon.  You "
"also need to execute semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH' "
"to add this\n"
"    new path to the system defaults. If you did not intend to use "
"$TARGET_PATH as a qemu\n"
"    image it could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил доступ qemu к $TARGET_PATH.\n"
"    Если это образ виртуализации, он должен иметь метку virt_image_t.\n"
"    Система настроена так, чтобы файлы образов в каталоге /var/lib/libvirt/"
"images\n"
"    отмечались корректно. Рекомендуется скопировать образ в этот каталог.\n"
"    Если вы действительно хотите работать с образами qemu в выбранном "
"каталоге,\n"
"    можно изменить метку $TARGET_PATH на virt_image_t,\n"
"    используя chcon. После этого понадобится выполнить команду\n"
"    «semanage fcontext -a -t virt_image_t '$FIX_TARGET_PATH'», \n"
"    чтобы добавить новый путь в системный список. Если вы не планировали "
"использовать\n"
"    $TARGET_PATH как образ qemu, возможно, это ошибка или попытка вторжения.\n"
"    "

#: ../src/qemu_file_image.py:48
msgid "If $TARGET_BASE_PATH is a virtualization target"
msgstr ""

#: ../src/restorecon.py:57
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.  restorecon '$TARGET_PATH', if this file is a "
"directory,\n"
"    you can recursively restore using restorecon -R '$TARGET_PATH'.\n"
"    "
msgstr ""
"\n"
"   Исходный контекст файла можно восстановить:\n"
"\n"
"    «restorecon '$TARGET_PATH'»\n"
"\n"
"   Для каталога используйте команду «restorecon -R '$TARGET_PATH'».\n"
"    "

#: ../src/restorecon.py:64
#, python-format
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $TARGET_PATH may\n"
"    be mislabeled.  $TARGET_PATH default SELinux type is\n"
"    <B>%s</B>, but its current type is <B>$TARGET_TYPE</B>. Changing\n"
"    this file back to the default type may fix your problem.\n"
"    <p>\n"
"    File contexts can be assigned to a file in the following ways.\n"
"    <ul>\n"
"        <li>Files created in a directory receive the file context of the "
"parent directory by default.\n"
"        <li>The SELinux policy might override the default label inherited "
"from the parent directory by\n"
"            specifying a process running in context A which creates a file "
"in a directory labeled B\n"
"            will instead create the file with label C. An example of this "
"would be the dhcp client running\n"
"            with the dhclient_t type and creating a file in the directory /"
"etc. This file would normally\n"
"            receive the etc_t type due to parental inheritance but instead "
"the file\n"
"            is labeled with the net_conf_t type because the SELinux policy "
"specifies this.\n"
"        <li>Users can change the file context on a file using tools such as "
"chcon, or restorecon.\n"
"    </ul>\n"
"    This file could have been mislabeled either by user error, or if an "
"normally confined application\n"
"    was run under the wrong domain.\n"
"    <p>\n"
"    However, this might also indicate a bug in SELinux because the file "
"should not have been labeled\n"
"    with this type.\n"
"    <p>\n"
"    If you believe this is a bug, please file a bug report against this "
"package.\n"
"    "
msgstr ""

#: ../src/restorecon.py:90
#, python-format
msgid ""
"If you want to fix the label. \n"
"$TARGET_PATH default label should be %s."
msgstr ""

#: ../src/restorecon.py:96
msgid ""
"you can run restorecon. The access attempt may have been stopped due to "
"insufficient permissions to access a parent directory in which case try to "
"change the following command accordingly."
msgstr ""

#: ../src/restorecon.py:105 ../src/restorecon_source.py:73
msgid "Restore\n"
"Context"
msgstr "Восстановить\n"
"контекст."

#: ../src/restorecon_source.py:39
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.  restorecon '$SOURCE_PATH'.\n"
"    "
msgstr ""
"\n"
"   Исходный контекст можно восстановить с помощью команды restorecon:\n"
"   restorecon -R  '$SOURCE_PATH'.\n"
"\n"
"    "

#: ../src/restorecon_source.py:45
#, python-format
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $SOURCE_PATH may\n"
"    be mislabeled.  $SOURCE_PATH default SELinux type is\n"
"    <B>%s</B>, but its current type is <B>$SOURCE_TYPE</B>. Changing\n"
"    this file back to the default type may fix your problem.\n"
"    <p>\n"
"    This file could have been mislabeled either by user error, or if an "
"normally confined application\n"
"    was run under the wrong domain.\n"
"    <p>\n"
"    However, this might also indicate a bug in SELinux because the file "
"should not have been labeled\n"
"    with this type.\n"
"    <p>\n"
"    If you believe this is a bug, please file a bug report against this "
"package.\n"
"    "
msgstr ""

#: ../src/restorecon_source.py:60
#, python-format
msgid ""
"If you want to fix the label. \n"
"$SOURCE_PATH default label should be %s."
msgstr ""

#: ../src/restorecon_source.py:65
msgid "you can run restorecon."
msgstr "можно выполнить restorecon."

#: ../src/rsync_data.py:31
msgid ""
"\n"
"    SELinux denied RSYNC access to $TARGET_PATH.\n"
"    If this is an RSYNC repository, it has to have a file context label of\n"
"    rsync_data_t. If you did not intend to use $TARGET_PATH as an RSYNC "
"repository,\n"
"    this message could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/rsync_data.py:38
msgid ""
"\n"
"    You can alter the file context by executing chcon -R -t rsync_data_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"rsync_data_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Файловый контекст можно изменить с помощью команды\n"
"    «chcon -R -t rsync_data_t '$TARGET_PATH'»\n"
"    Также необходимо изменить контексты файлов в системе, \n"
"    используемые по умолчанию, \n"
"    чтобы они сохранились даже при повторной разметке. \n"
"    «semanage fcontext -a -t rsync_data_t '$FIX_TARGET_PATH'»\n"
"    "

#: ../src/rsync_data.py:45
msgid "If $TARGET_BASE_PATH should be shared via the RSYNC daemon"
msgstr ""

#: ../src/rsync_data.py:46
msgid "You need to change the label on $TARGET_BASE_PATH"
msgstr "Необходимо изменить метку для $TARGET_BASE_PATH"

#: ../src/samba_share.py:27
msgid ""
"\n"
"    SELinux is preventing Samba ($SOURCE_PATH) \"$ACCESS\" access to "
"$TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает Samba ($SOURCE_PATH) получить доступ «$ACCESS» к "
"$TARGET_PATH. \n"
"    "

#: ../src/samba_share.py:31
msgid ""
"\n"
"    SELinux denied samba access to $TARGET_PATH.\n"
"    If you want to share this directory with samba it has to have a file "
"context label of\n"
"    samba_share_t. If you did not intend to use $TARGET_PATH as a samba "
"repository,\n"
"    this message could indicate either a bug or an intrusion attempt.\n"
"    Please refer to 'man samba_selinux' for more information on setting up "
"Samba and SELinux.\n"
"    "
msgstr ""

#: ../src/samba_share.py:39
msgid ""
"\n"
"    You can alter the file context by executing chcon -R -t samba_share_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"samba_share_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Контекст файла можно изменить, выполнив команду\n"
"    «chcon -R -t samba_share_t '$TARGET_PATH'»\n"
"    Также необходимо изменить контексты файлов в системе, \n"
"    используемые по умолчанию, \n"
"    чтобы они сохранились даже при повторной разметке. \n"
"    «semanage fcontext -a -t samba_share_t '$FIX_TARGET_PATH'»\n"
"    "

#: ../src/samba_share.py:53
#, python-format
msgid ""
"# semanage fcontext -a -t samba_share_t '$FIX_TARGET_PATH%s'\n"
"# restorecon %s -v '$FIX_TARGET_PATH'"
msgstr ""
"# semanage fcontext -a -t samba_share_t '$FIX_TARGET_PATH%s'\n"
"# restorecon %s -v '$FIX_TARGET_PATH'"

#: ../src/sandbox_connect.py:33
msgid ""
"\n"
"    SELinux has denied $SOURCE from connecting to a network port "
"$PORT_NUMBER within a sandbox.\n"
"    If $SOURCE should be allowed to connect on $PORT_NUMBER, you need to use "
"a different sandbox type like sandbox_web_t or sandbox_net_t.  \n"
"    # sandbox -X -t sandbox_net_t $SOURCE\n"
"    \n"
"\n"
"If $SOURCE is not supposed\n"
"    to connect to $PORT_NUMBER, this could signal an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/sandbox_connect.py:41
msgid ""
"\n"
"    If you want to allow $SOURCE to connect to $PORT_NUMBER, you can execute "
"\n"
"\n"
"    # sandbox -X -t sandbox_net_t $SOURCE\n"
"    "
msgstr ""
"\n"
"    Чтобы разрешить подключение $SOURCE к порту $PORT_NUMBER, выполните:\n"
"\n"
"    # sandbox -X -t sandbox_net_t $SOURCE\n"
"    "

#: ../src/sandbox_connect.py:49
msgid ""
"you need to modify the sandbox type. sandbox_web_t or sandbox_net_t.\n"
"For example:\n"
"sandbox -X -t sandbox_net_t $SOURCE_PATH\n"
"Please read 'sandbox' man page for more details.\n"
msgstr ""
"необходимо изменить type. sandbox_web_t или sandbox_net_t.\n"
"Пример:\n"
"sandbox -X -t sandbox_net_t $SOURCE_PATH\n"
"За подробной информацией обратитесь к справочной странице sandbox.\n"

#: ../src/selinuxpolicy.py:29
msgid "\n"
"    Your system may be seriously compromised!\n"
"    "
msgstr "\n"
"    Ваша система может быть серьёзно скомпрометирована!\n"
"    "

#: ../src/selinuxpolicy.py:33
msgid ""
"\n"
"    SELinux has prevented $SOURCE from modifying $TARGET.  This denial\n"
"    indicates $SOURCE was trying to modify the selinux policy configuration.\n"
"    All applications that need this access should have already had policy\n"
"    written for them.  If a compromised application tries to modify the "
"SELinux\n"
"    policy this AVC will be generated. This is a serious issue. Your system\n"
"    may very well be compromised.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил $SOURCE модифицировать $TARGET.\n"
"    $SOURCE пытался изменить конфигурацию политики selinux.\n"
"    Для всех приложений, которым необходим такой уровень доступа,\n"
"    должна существовать отдельная политика.\n"
"    Если скомпрометированное приложение попытается изменить политику,\n"
"    будет сгенерировано AVC.\n"
"    Это является серьезной проблемой и может служить индикатором того, \n"
"    что система могла быть скомпрометирована.\n"
"    "

#: ../src/setenforce.py:27
msgid "If you did not directly cause this AVC through testing."
msgstr ""

#: ../src/setenforce.py:28
msgid "if you think that you might have been hacked"
msgstr "вы считаете, что вы стали жертвой взлома."

#: ../src/setenforce.py:32
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH tried to modify "
"SELinux enforcement.\n"
"    "
msgstr ""
"\n"
"    Ваша система может быть серьёзно скомпрометирована! $SOURCE_PATH "
"пытается изменить принудительный режим SELinux.\n"
"    "

#: ../src/setenforce.py:36
msgid ""
"\n"
"    SELinux has prevented $SOURCE from writing to a file under /sys/fs/"
"selinux.\n"
"    Files under /sys/fs/selinux control the way SELinux is configured.\n"
"    All programs that need to write to files under /sys/fs/selinux should "
"have already had policy\n"
"    written for them. If a compromised application tries to turn off SELinux\n"
"    this AVC will be generated. This is a serious issue. Your system may "
"very\n"
"    well be compromised.\n"
"    "
msgstr ""

#: ../src/setenforce.py:48
msgid "If you believe $SOURCE_PATH tried to disable SELinux."
msgstr ""

#: ../src/setenforce.py:49
msgid ""
"you may be under attack by a hacker, since confined applications should "
"never need this access."
msgstr ""
"возможно, вы подверглись хакерской атаке, так как защищенные приложения не "
"должны запрашивать этот уровень доступа."

#: ../src/sshd_root.py:36
msgid ""
"\n"
"    SELinux denied access requested by $SOURCE. $TARGET_PATH may\n"
"    be mislabeled. sshd is allowed to read content in /root/.ssh directory "
"if it\n"
"    is labeled correctly.\n"
"    "
msgstr ""

#: ../src/sshd_root.py:42
msgid ""
"\n"
"    You can restore the default system context to this file by executing the\n"
"    restorecon command.\n"
"    # restorecon -R /root/.ssh\n"
"    "
msgstr ""

#: ../src/sshd_root.py:55
msgid "Restore Context"
msgstr "Восстановить контекст"

#: ../src/swapfile.py:27
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH \"$ACCESS\" to $TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает $SOURCE_PATH доступ «$ACCESS» к $TARGET_PATH.\n"
"    "

#: ../src/swapfile.py:31
msgid ""
"\n"
"    SELinux denied $SOURCE access to $TARGET_PATH.\n"
"    If this is a swapfile, it has to have a file context label of\n"
"    swapfile_t. If you did not intend to use\n"
"    $TARGET_PATH as a swapfile, this message could indicate either a bug or "
"an intrusion attempt.\n"
"    "
msgstr ""

#: ../src/swapfile.py:38
msgid ""
"\n"
"    You can alter the file context by executing chcon -t swapfile_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"swapfile_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Файловый контекст можно изменить, выполнив\n"
"    «сhcon -t swapfile_t '$TARGET_PATH'»\n"
"    Также необходимо изменить контексты файлов в системе, \n"
"    используемые по умолчанию, \n"
"    чтобы они сохранились даже при повторной разметке. \n"
"    «semanage fcontext -a -t swapfile_t '$FIX_TARGET_PATH'»\n"
"    "

#: ../src/sys_module.py:28
msgid ""
"If you do not believe your $SOURCE_PATH should be modifying the kernel, by "
"loading kernel modules"
msgstr ""

#: ../src/sys_module.py:29
msgid "You might have been hacked."
msgstr "Возможно, вы стали жертвой взлома."

#: ../src/sys_module.py:33
msgid ""
"\n"
"    Your system may be seriously compromised! $SOURCE_PATH tried to load a "
"kernel module.\n"
"    "
msgstr ""
"\n"
"    Ваша система может быть серьёзно скомпрометирована! $SOURCE_PATH "
"пытается загрузить модуль ядра.\n"
"    "

#: ../src/sys_module.py:37
msgid ""
"\n"
"    SELinux has prevented $SOURCE from loading a kernel module.\n"
"    All confined programs that need to load kernel modules should have "
"already had policy\n"
"    written for them. If a compromised application\n"
"    tries to modify the kernel this AVC will be generated. This is a serious\n"
"    issue. Your system may very well be compromised.\n"
"    "
msgstr ""
"\n"
"    SELinux запретил $SOURCE загружать модуль ядра.\n"
"    Для всех приложений, которым необходим такой уровень доступа,\n"
"    должна существовать отдельная политика.\n"
"    Если скомпрометированное приложение попытается изменить модуль,\n"
"    будет сгенерировано AVC.\n"
"    Это является серьёзной проблемой и может служить индикатором того, \n"
"    что система была скомпрометирована.\n"
"     "

#: ../src/sys_module.py:46
msgid ""
"If you do not believe that $SOURCE_PATH should be attempting to modify the "
"kernel by loading a kernel module."
msgstr ""

#: ../src/sys_module.py:47
msgid "A process might be attempting to hack into your system."
msgstr "Процесс пытается осуществить несанкционированный доступ к системе."

#: ../src/sys_resource.py:28
msgid ""
"\n"
"    SELinux is preventing $SOURCE_PATH the \"sys_resource\" capability.\n"
"    "
msgstr "\n"
"   SELinux запрещает возможность $SOURCE_PATH «sys_resource».\n"
"    "

#: ../src/sys_resource.py:32
msgid ""
"\n"
"    Confined domains should not require \"sys_resource\". This usually means "
"that     your system is running out some system resource like disk space, "
"memory, quota etc. Please clear up the disk and this\n"
"    AVC message should go away. If this AVC continues after you clear up the "
"disk space, please report this as a bug.\n"
"    "
msgstr ""
"\n"
"Ограниченные домены не должны требовать «sys_resource». Обычно это означает, "
"что не хватает какого-либо ресурса, например, места на диске, памяти, квоты "
"и т.д. Освободите место на диске и это \n"
"AVC-сообщение должно исчезнуть. Если проблема сохранится после очистки "
"диска, создайте отчет об ошибке.\n"
"    "

#: ../src/sys_resource.py:39
msgid ""
"If you do not want processes to require capabilities to use up all the "
"system resources on your system;"
msgstr ""

#: ../src/sys_resource.py:40
msgid ""
"you need to diagnose why your system is running out of system resources and "
"fix the problem.\n"
"\n"
"According to /usr/include/linux/capability.h, sys_resource is required to:\n"
"\n"
"/* Override resource limits. Set resource limits. */\n"
"/* Override quota limits. */\n"
"/* Override reserved space on ext2 filesystem */\n"
"/* Modify data journaling mode on ext3 filesystem (uses journaling\n"
"   resources) */\n"
"/* NOTE: ext2 honors fsuid when checking for resource overrides, so\n"
"   you can override using fsuid too */\n"
"/* Override size restrictions on IPC message queues */\n"
"/* Allow more than 64hz interrupts from the real-time clock */\n"
"/* Override max number of consoles on console allocation */\n"
"/* Override max number of keymaps */\n"
msgstr ""

#: ../src/vbetool.py:30
msgid ""
"\n"
"    SELinux has prevented vbetool from performing an unsafe memory operation."
"\n"
"    "
msgstr ""
"\n"
"    SELinux предотвратил выполнение программой vbetool небезопасной операции "
"с памятью.\n"
"    "

#: ../src/vbetool.py:34
msgid ""
"\n"
"SELinux denied an operation requested by $SOURCE, a program used\n"
"to alter video hardware state.  This program is known to use\n"
"an unsafe operation on system memory but so are a number of\n"
"malware/exploit programs which masquerade as vbetool.  This tool is used to\n"
"reset video state when a machine resumes from a suspend.  If your machine\n"
"is not resuming properly your only choice is to allow this\n"
"operation and reduce your system security against such malware.\n"
"\n"
"    "
msgstr ""
"\n"
"SELinux запретила действие, запрошенное $SOURCE, программой изменения\n"
"состояния видео оборудования. Эта программа обычно выполняет небезопасные\n"
"операции памяти, в то же время именно так поступают и вредоносные программы,\n"
"замаскированные под vbetool.\n"
"Эта утилита восстанавливает состояние видео при возврате системы из\n"
"ждущего режима. Если компьютер не возобновляет работу, остаётся только\n"
"разрешить это действие, тем самым ослабив защиту системы.\n"
"\n"
"    "

#: ../src/vbetool.py:45 ../src/wine.py:56
msgid ""
"\n"
"If you decide to continue to run the program in question you will need\n"
"to allow this operation.  This can be done on the command line by\n"
"executing:\n"
"\n"
"# setsebool -P mmap_low_allowed 1\n"
msgstr ""
"\n"
"Если вы хотите продолжить выполнение данной программы, необходимо\n"
"это разрешить. Для этого выполните:\n"
"\n"
"# setsebool -P mmap_low_allowed 1\n"

#: ../src/vbetool.py:55
msgid ""
"If you want to ignore this AVC because it is dangerous and your machine "
"seems to be working correctly."
msgstr ""

#: ../src/vbetool.py:56
msgid ""
"you must tell SELinux about this by enabling the vbetool_mmap_zero_ignore "
"boolean."
msgstr ""
"Установите переменную vbetool_mmap_zero_ignore, чтобы сообщить об этом "
"SELinux."

#: ../src/vbetool.py:62 ../src/wine.py:73
msgid "Turn off memory protection"
msgstr "Выключить защиту памяти"

#: ../src/wine.py:31
msgid ""
"\n"
"    SELinux has prevented wine from performing an unsafe memory operation.\n"
"    "
msgstr ""
"\n"
"    SELinux предотвратил выполнение программой wine небезопасной операции с "
"памятью.\n"
"    "

#: ../src/wine.py:35
msgid ""
"\n"
"SELinux denied an operation requested by wine-preloader, a program used\n"
"to run Windows applications under Linux.  This program is known to use\n"
"an unsafe operation on system memory but so are a number of\n"
"malware/exploit programs which masquerade as wine.  If you were\n"
"attempting to run a Windows program your only choices are to allow this\n"
"operation and reduce your system security against such malware or to\n"
"refrain from running Windows applications under Linux.  If you were not\n"
"attempting to run a Windows application this indicates you are likely\n"
"being attacked by some for of malware or program trying to exploit your\n"
"system for nefarious purposes.\n"
"\n"
"Please refer to\n"
"\n"
"http://wiki.winehq.org/PreloaderPageZeroProblem\n"
"\n"
"Which outlines the other problems wine encounters due to its unsafe use\n"
"of memory and solutions to those problems.\n"
"\n"
"    "
msgstr ""
"\n"
"SELinux запретил выполнение операции, запрошенной wine-preloader, программой "
"для запуска приложений Windows на Linux. Известно, что эта программа "
"использует небезопасные операции с системной памятью, как и вредоносные "
"программы, маскирующиеся под wine. Если вы\n"
"пытаетесь запустить приложение Windows, можно либо разрешить\n"
"данную операцию, тем самым ослабив защиту системы,\n"
"либо отказаться от использования приложений Windows под Linux. Если же вы не "
"пытались запустить приложение Windows, это означает, что вы атакованы одной "
"из таких вредоносных программ или кто-то пытается использовать вашу систему.\n"
"\n"
"Обратитесь за более подробной информацией к статье \n"
"\n"
"http://wiki.winehq.org/PreloaderPageZeroProblem\n"
"\n"
"Там описаны другие проблемы, с которыми сталкивается программа wine из-за "
"небезопасных операций с памятью, и возможные пути их решения.\n"
"\n"
"    "

#: ../src/wine.py:66
msgid ""
"If you want to ignore this AVC because it is dangerous and your wine "
"applications are working correctly."
msgstr ""

#: ../src/wine.py:67
msgid ""
"you must tell SELinux about this by enabling the wine_mmap_zero_ignore "
"boolean."
msgstr ""
"установите переменную wine_mmap_zero_ignore, чтобы сообщить об этом SELinux."

#: ../src/xen_image.py:27
msgid ""
"\n"
"    SELinux is preventing xen ($SOURCE_PATH) \"$ACCESS\" access to "
"$TARGET_PATH.\n"
"    "
msgstr ""
"\n"
"    SELinux запрещает xen ($SOURCE_PATH) доступ «$ACCESS» к $TARGET_PATH. \n"
"    "

#: ../src/xen_image.py:31
msgid ""
"\n"
"    SELinux denied xen access to $TARGET_PATH.\n"
"    If this is a XEN image, it has to have a file context label of\n"
"    xen_image_t. The system is setup to label image files in directory /var/"
"lib/xen/images\n"
"    correctly.  We recommend that you copy your image file to /var/lib/xen/"
"images.\n"
"    If you really want to have your xen image files in the current "
"directory, you can relabel $TARGET_PATH to be xen_image_t using chcon.  You "
"also need to execute semanage fcontext -a -t xen_image_t '$FIX_TARGET_PATH' "
"to add this\n"
"    new path to the system defaults. If you did not intend to use "
"$TARGET_PATH as a xen\n"
"    image it could indicate either a bug or an intrusion attempt.\n"
"    "
msgstr ""
"\n"
"    SELinux закрыл доступ xen к $TARGET_PATH.\n"
"    Если это образ XEN, он должен иметь метку контекста\n"
"    xen_image_t. Система настроена так, чтобы файлы образов в каталоге /var/"
"lib/xen/images\n"
"    отмечались корректно. Рекомендуется скопировать образ в этот каталог.\n"
"    Если вы действительно хотите работать с образами xen в выбранном "
"каталоге,\n"
"    можно изменить метку $TARGET_PATH на xen_image_t,\n"
"    используя chcon. После этого понадобится выполнить команду\n"
"    «semanage fcontext -a -t xen_image_t '$FIX_TARGET_PATH'», чтобы добавить "
"новый путь\n"
"    в системный список. Если вы не планировали использовать\n"
"    $TARGET_PATH как образ xen, возможно, это ошибка или попытка вторжения.\n"
"    "

#: ../src/xen_image.py:41
msgid ""
"\n"
"    You can alter the file context by executing chcon -t xen_image_t "
"'$TARGET_PATH'\n"
"    You must also change the default file context files on the system in "
"order to preserve them even on a full relabel.  \"semanage fcontext -a -t "
"xen_image_t '$FIX_TARGET_PATH'\"\n"
"    "
msgstr ""
"\n"
"    Файловый контекст можно изменить, выполнив команду\n"
"    «chcon -t xen_image_t '$TARGET_PATH'»\n"
"    Также необходимо изменить контексты файлов в системе, \n"
"    используемые по умолчанию, \n"
"    чтобы они сохранились даже при повторной разметке. \n"
"    «semanage fcontext -a -t xen_image_t '$FIX_TARGET_PATH'»\n"
"    "
